Dockerfile

```diff
@@ -30,6 +30,15 @@ RUN \
         \
         && rm -rf /var/lib/apt/lists/*
 
+# Install Julia from the official website
+
+RUN curl https://julialang-s3.julialang.org/bin/linux/x64/1.0/julia-1.0.1-linux-x86_64.tar.gz --output julia-1.0.1-linux-x86_64.tar.gz
+RUN echo '9ffbcf7f4a111e13415954caccdd1ce90b5c835cee9f62d6ac708f5b752c87dd  julia-1.0.1-linux-x86_64.tar.gz' > .sha256
+RUN sha256sum -c .sha256
+RUN rm .sha256
+RUN tar xvf julia-1.0.1-linux-x86_64.tar.gz -C /root
```

Comment:
For Docker space-saving reasons, downloading the tarball and deleting it need to be in a single `RUN` statement. Something like:

```
RUN \
        curl \
        https://julialang-s3.julialang.org/bin/linux/x64/1.0/julia-1.0.1-linux-x86_64.tar.gz \
        --output julia-1.0.1-linux-x86_64.tar.gz \
        \
        && echo '9ffbcf7f4a111e13415954caccdd1ce90b5c835cee9f62d6ac708f5b752c87dd  julia-1.0.1-linux-x86_64.tar.gz' > .sha256 \
        && sha256sum -c .sha256 \
        && rm .sha256 \
        && tar xvf julia-1.0.1-linux-x86_64.tar.gz -C /root \
        && rm julia-1.0.1-linux-x86_64.tar.gz \
```

I hate it, but it's easier than having separate build files or using build stages.

Reply:
That was exactly what I was about to try next! It'll be but a moment...